reddit_service_websockets/socketserver.py

```diff
@@ -46,6 +45,16 @@ def log_request(self):
         # geventwebsocket logs every request at INFO level. that's annoying.
         pass
 
+    @property
+    def socket_server(self):
+        # this is a pretty nasty smell. we're violating the wsgi model by
+        # expecting the application callable to be of a specific type so that
+        # we can get access to its state for various things (metrics, secrets).
+        # this property at least gets us a little encapsulation.
+        socket_server = self.application.application
```

Comment:
I'm confused by the second level of accessing here (`application.application`), how does this work?